### DISCUSSION TITLE
安装第三方库（ts-md5），导致项目出错

### DISCUSSION THREAD

**Discussion by luckyxiaomo** [ant-design/ant-design#31714] [Q&A] [answered]
为了给密码加密，安装了ts-md5后，执行npm run dev 在浏览器下报错

9mf-va_remoteEntry.js:744 Uncaught (in promise) Error: Module "./ts-md5" does not exist in container.
while loading "./ts-md5" from webpack/container/reference/mf
    at mf-va_remoteEntry.js:744

请问有什么特殊的使用方法吗？

  "name": "ant-design-pro",
  "version": "5.0.0",
  "private": true,
  "description": "An out-of-box UI solution for enterprise applications",

**Comment by yoyo837** [comment]
你的问题和该项目无关吧, 或者提供更多信息.

**Comment by luckyxiaomo** [comment, marked as answer]
多次重复安装就没这个错误提示了

在有这个错误的情况下，编译的正式版也没有这个错误提示
